File: adk/tests/fixtures/unit_profiles/java-junit/project/src/main/java/com/example/Calculator.java
package com.example;

public final class Calculator {
    private Calculator() {}

    public static int add(int left, int right) {
        return left + right;
    }

    public static double divide(double dividend, double divisor) {
        if (divisor == 0) {
            throw new IllegalArgumentException("divisor must not be zero");
        }
        return dividend / divisor;
    }
}

File: adk/tests/fixtures/unit_profiles/go-testing/project/calculator.go
package calculator

import "errors"

func Add(left, right int) int {
	return left + right
}

func Divide(dividend, divisor float64) (float64, error) {
	if divisor == 0 {
		return 0, errors.New("divisor must not be zero")
	}
	return dividend / divisor, nil
}
